packages/uswds-utilities/src/styles/rules/border.scss

```diff
@@ -12,11 +12,11 @@ output:
   border-[modifier]: [value];
 ----------------------------------------
 example:
-  .border-t-1px {
+  .border-top-1px {
     border-top: 1px solid; }
 
   .border-0 {
-    border: none; }
+    border: 0 solid; }
```

Comment:
```suggestion
    border: 0 solid; 
  }
```

Reply:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)